darts/models/forecasting/regression_ensemble_model.py

```diff
@@ -69,6 +92,52 @@ def __init__(
             f"{regression_model.lags}",
         )
 
+        raise_if(
```

Comment:
Can we move all those tests to the base class?

Reply:
Sure, there will be a small discrepancies since the name of the argument/attributes are slightly different between the two classes (`regression_` is used as a prefix in `RegressionEnsembleModel`)